### ISSUE TITLE
Live vms url functinalities not working

### ISSUE THREAD

**Issue by Kajol-Kumari** [anitab-org/vms#1041]
## Description
The vms is live [here](http://ec2-52-53-177-18.us-west-1.compute.amazonaws.com/en-us/). But none of its functionalities are working except the home page UI.

### Update [Required]
- [ ] Fix the live vms so that we can get all of it's functionlities working.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
48 hours

**Comment by ashwinijha6** [comment]
Hey Hi @Kajol-Kumari I would like to work on this issue as an OSH participant

**Comment by Kajol-Kumari** [comment]
Hey @ashwinijha6 glad to see that you are interested in this issue. We will assign it to you, once the `OSH` event starts :)

**Comment by ashwinijha6** [comment]
Thanku @Kajol-Kumari looking forward to working on it :)

**Comment by SanketDG** [comment]
The deployment is from outdated code, so there's a chance it may not run at all. I think we need to move to a much more reliable and easy to go platform, but this is ultimately in the hands of maintainers and admins.

**Comment by Kajol-Kumari** [comment]
> The deployment is from outdated code, so there's a chance it may not run at all. I think we need to move to a much more reliable and easy to go platform, but this is ultimately in the hands of maintainers and admins.

Yes @SanketDG totally agree as the maintainers and admin needs to share the details of the Ec2 instance or the other platform where we decide to make this project live and this is the reason i have added it under `outreach`.

**Comment by ashwinijha6** [comment]
@Kajol-Kumari could you please elaborate about the issue

**Comment by Kajol-Kumari** [comment]
> @Kajol-Kumari could you please elaborate about the issue

Hey @ashwinijha6 So this issue is more of a discussion on deciding the platform to make the vms project live as the current [ulr](http://ec2-52-53-177-18.us-west-1.compute.amazonaws.com/en-us/) for the same have non of the functionlities working and it's in the hand of maintainers and administrators to decide where they want to host it. So, what asignee can do is make a blog post explaining his/her approach of making the project live along with the reason of choosing the particular platform for hosting this project.

\cc: @SanketDG

**Comment by Kajol-Kumari** [comment]
Hey Everybody, all who are interested in this issue. Please accept my apologies but according to the new guidelines we received, we will count `First come first serve` on the basis of comments/claims done 1st september onwards. So, everyone who are interested in this issue, do come again on 1st and comment first to get assigned.

**Comment by ashwinijha6** [comment]
@Kajol-Kumari  I would like to work on this as an OSH participant

**Comment by Kajol-Kumari** [comment]
Hey @ashwinijha6 please go ahead with this one :+1: 
please make sure you have gone through [this comment](https://github.com/anitab-org/vms/issues/1041#issuecomment-682965032) and issue description properly before starting.
Happy Learning :)

**Comment by ashwinijha6** [comment]
yes @Kajol-Kumari looking forward for ur help & guidance :)

**Comment by Kajol-Kumari** [comment]
@ashwinijha6 can you please provide the update on you work progress for this issue.

**Comment by ashwinijha6** [comment]
@Kajol-Kumari  I need some time as I was working on another issue

**Comment by SanketDG** [comment]
@ashwinijha6 Please open this issue to others if you are busy working on something else. The general rule in OSH is to work one issue at a time.

**Comment by ashwinijha6** [comment]
ok @SanketDG  you can make it available for others

**Comment by psingh587** [comment]
Is this issue up for grabs? @SanketDG

**Comment by priyanshuone6** [comment]
Hey, can you please assign this issue to me

**Comment by Kajol-Kumari** [comment]
@psingh587 I can see you have 2 issues in your hand as of now so i am assigning it to @priyanshuone6 . @priyanshuone6 please make sure to open ip a PR or atleast provide some status of your work within 3 days.

**Comment by priyanshuone6** [comment]
Till now I wrote about my approach on how and where to host the vms but I am getting confused on what should I write to fix the live vms, can you please elaborate more on it
